Gruntfile.js

```diff
@@ -60,29 +58,46 @@ module.exports = function (grunt) {
 				/** list of files to ignore **/
 				ignorefile: '.retireignore.json' //or '.retireignore.json'
 			},
-			js: ['lib/*.js'], /** Which js-files to scan. **/
-			node: ['./'] /** Which node directories to scan (containing package.json). **/
+			js: ['lib/*.js'] /** Which js-files to scan. **/,
+			node: [
+				'./'
+			] /** Which node directories to scan (containing package.json). **/
 		},
 		clean: ['dist', 'tmp', 'axe.js', 'axe.*.js'],
 		babel: {
 			options: {
 				compact: 'false'
 			},
 			core: {
-				files: [{
-					expand: true,
-					cwd: 'lib/core',
-					src: ['**/*.js'],
-					dest: 'tmp/core'
-				}]
+				files: [
+					{
```

Comment:
a whole line for a semicolon is a waste of space

Reply:
This is just how Prettier formats arrays of objects. Unfortunately there's not much we can do to change this.